src/main/java/frc/robot/commands/MoveArm.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.robot.subsystems.Drivetrain;
+
+public class MoveArm extends CommandBase{
```

Comment:
What is this for?

Reply:
Basic command structure for how to use selectedNode.

Reply:
We don't need to write command structure like this